src/averageAges.js

```diff
@@ -20,6 +20,17 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  let person = people.filter(man => man.sex === 'm');
+
+  person = (century === undefined)
```

Comment:
1. simplify condition
2. handle truth case forst
```suggestion
  person = century ? ... : ...
```